docs/src/components/ColorContrastHelpers/ColorContrastHelpers.component.tsx

```diff
@@ -0,0 +1,32 @@
+// VENDOR
+import * as wcagContrast from 'wcag-contrast';
+// THEME
+import { colors } from '../../../../src/theme';
+
+
+export class ColorContrastHelpers {
+
+  public static getRoundedColorContrast(colorHexCode1, colorHexCode2): number {
+    let exactContrast = wcagContrast.hex(colorHexCode1, colorHexCode2);
+    return Math.round(exactContrast * 100) / 100;
+  }
+
+  public static getSuccessOrErrorColor(colorContrast): number {
+    if (colorContrast > 4.5) {
+      return colors.success.base;
+    } else {
+      return colors.error.base;
+    }
+  }
+
+  public static getHigherContrastColor(colorHexCode): string {
+    let contrastOnBlack = wcagContrast.hex('#000', colorHexCode);
```

Comment:
I think I was using `let` because I was changing the value later. I refactored this one as well to not have any variables. I did update to use named colors rather than hex codes.